src/eventsAfter.ts

```diff
@@ -34,14 +34,14 @@ export async function eventsAfter (args: string[]): Promise<void> {
 
 async function getMessage (hash: string): Promise<ReturnType<typeof JSON.parse>> {
   const value = await readString(hash)
-  if (!value) throw new Error(`no entry for ${hash}!`)
+  if (value === undefined || value === null) throw new Error(`no entry for ${hash}!`)
```

Comment:
Same request for reverting these (and all other ones, I'll stop repeating myself, but please check the rest of the codebase for this)